lib/order_query/point.rb

```diff
@@ -68,14 +68,14 @@ def value(column)
       v = record.send(column.name)
       if v.nil? && !column.nullable?
         fail Errors::NonNullableColumnIsNullError,
-             "Column #{column.inspect} is NULL on record #{@record.inspect}. "\
+             "Column #{column.inspect} is NULL on record #{@record.send(inspect_method)}. "\
```

Comment:
Calling a `private` method using `send` feels a bit fishy; perhaps it makes sense to make the method `protected` and using as usual?